ocean_lib/data_provider/test/test_base.py

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright 2022 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+from ocean_lib.data_provider.base import DataServiceProviderBase
+
+
+def test_sanitize_content_disposition():
+    header = "./../../my/relative/path"
+    res = DataServiceProviderBase._sanitize_content(header)
+    assert res is False
```

Comment:
you can `assert` res and `assert not res`, "assert ... is True" is superfluous. I would also cover the _get_file_name statements.